docs/api/ShallowWrapper/rightContains.md

```diff
@@ -0,0 +1,64 @@
+# `.rightContains(nodeOrNodes) => Boolean`
+
+Returns whether or not the current wrapper has a node anywhere in it's render tree that looks like
+the one passed in. Equality is based on the expected element (`nodeOrNodes`) and not on the wrapper. It will determine if an element in the wrapper `looks like` the expected element by checking if all props of the expected element are present on the wrappers element and equals to each other.
+
+
+#### Arguments
+
+1. `nodeOrNodes` (`ReactElement|Array<ReactElement>`): The node or array of nodes whose presence you are detecting in the current instance's
```

Comment:
I don't think that's accurate. `shallow(<A a><B b /></A>).containsMatchingElement(<B />)` isn't the same thing as `shallow(<B />).contains(<A a><B b /></A>)`.

I see this as simply "contains", but using the element-to-be-found's prop list rather than the element-being-searched.